gtsam/geometry/OrientedPlane3.h

```diff
@@ -24,6 +24,9 @@
 #include <gtsam/geometry/Unit3.h>
 #include <gtsam/geometry/Pose3.h>
 
+#include <algorithm>
```

Comment:
Move to cpp?

Reply:
not required any more, removing `#include`